sdk/core/azure-core-test/src/main/java/com/azure/core/test/TestBase.java

```diff
@@ -23,6 +28,12 @@
 public abstract class TestBase implements BeforeEachCallback {
     // Environment variable name used to determine the TestMode.
     private static final String AZURE_TEST_MODE = "AZURE_TEST_MODE";
+    private static final String AZURE_TEST_HTTP_CLIENTS = "AZURE_TEST_HTTP_CLIENTS";
+    protected static final String AZURE_TEST_HTTP_CLIENTS_VALUE_ALL = "ALL";
```

Comment:
protected and static are contradictory. Change this to public instead.

Reply:
Done